src/sql/workbench/browser/modelComponents/media/listView.css

```diff
@@ -0,0 +1,46 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+.listview-container {
+	display: flex;
+	flex-direction: column;
+	height: 100%;
+	max-width: 150px;
+	min-width: 120px;
+	font-size: 100%;
+	font-weight: inherit;
+	overflow: auto;
+	padding: 0 0 0 0;
+	font-size: 12px;
+}
+
+.listview-title {
```

Comment:
I'd like to see these all scoped down under listview-container and then also have that class name indicate that it's a modelview component for clarify - something like `modelview-listview-container`